### ISSUE TITLE
Jordan Harband onboarding

### ISSUE THREAD

**Issue by RafaelGSS** [nodejs/Release#911]
* [ ] Be added to the GitHub [releasers team](https://github.com/orgs/nodejs/teams/releasers) in the Node.js org (grants ci-release access)
* [ ] Be added to the GitHub [security-release team](https://github.com/orgs/nodejs/teams/security-release) in the Node.js and nodejs-private orgs
* [ ] Have a single, high quality SSH key added to the "dist" user on the primary www server (see below for guidelines regarding SSH key quality)
* [ ] Open a PR to add their GPG key to the nodejs/node [README.md](https://github.com/nodejs/node/#release-keys)
  * [ ] Once landed, the key should be cherry-picked to all active release staging branches (i.e. the active `vMM.x-staging` branches).
  * [ ] Any future updates to the key should also be cherry-picked to all active release staging branches.
* [ ] Open a PR in [nodejs/docker-node](https://github.com/nodejs/docker-node/) to add their GPG key to [node.keys](https://github.com/nodejs/docker-node/blob/main/keys/node.keys)
* [ ] Open a PR in [nodejs/release-keys](https://github.com/nodejs/release-keys) to add their GPG key via `./cli.sh add $KEY_ID`: 
* [ ] Open a PR in [nodejs/Release](https://github.com/nodejs/Release) to add themself under the "Releasers team" heading via `ncu-team sync README.md`.
* [ ] Add them to the `#nodejs-release-private` team on the OpenJS Slack.

cc: @ljharb

**Comment by RafaelGSS** [comment]
The next step is to schedule a pair release where Jordan will prepare the release and another releaser will sign it. As I said previously, I can't handle/sign any release this month. I can help with the first release of October. @nodejs/releasers feel free to take the lead on this onboarding if any of you can onboard Jordan sooner than me.

**Comment by ljharb** [comment]
Now that I'm a collaborator, I'd love to do this onboarding :-)

**Comment by ruyadorno** [comment]
Yes!! Now that the collaborator status is figured out let's get the onboarding started, we can def use the help in the Releases team 😊

**Comment by RafaelGSS** [comment]
Right, my suggestion is to participate in at least 2 releases meeting and then start preparing releases (manually)
